thoth/storages/graph/postgres.py

```diff
@@ -4822,6 +4894,57 @@ def sync_revsolver_result(self, document: Dict[str, Any]) -> None:
                     platform=entry["platform"],
                 )
 
+    def _check_package_solved(self, session: Session, package_name: str, package_version: str,
+                              package_index: str, os_name: str, os_version: str, python_version: str) -> Tuple:
```

Comment:
we do not require solver (os_name, os_version, python_version), we just want to know if it exists from any solver, because SI is not related to solver.